build_tools/ci/cpu_comparison/run.py

```diff
@@ -327,6 +327,8 @@ def _execute(self, config):
             atol=0,
             n_repeats=1,
             output_type=get_output_type(self.filename),
+            coalesce_loops=True,
```

Comment:
Can you add a comment with these truncf tests indicating why these flags are enabled? Something like, "currently without enabling loop coalescing (and dimension collapsing) we run out of program memory, this is under investigation" (similarly in other PR for outlining)